hologram_dashboard/fetcher/hologram_data_source_availability_fetcher.cc

```diff
@@ -55,16 +55,98 @@ void HologramDataSourceAvailabilityFetcher::FetchFromDatabase() {
 }
 
 void HologramDataSourceAvailabilityFetcher::GetHologramDataAvailability(
-    absl::Time date) {
-    // TODO(alexanderlin): Implement.
-    // for(const std::pair<std::string, std::string>& it : system_to_cell_map_) {
-    //     for(int i = 0; i < hologram_configs_.data_source_config_size(); ++i) {
-    //     UpdateProto(it.first, date, hologram_configs_.data_source_config(i), status);
-    // }
+    absl::Time time) {
+    absl::TimeZone google_time;
+    assert(absl::LoadTimeZone("America/Los_Angeles", &google_time));
```

Comment:
google_timezone can be a member variable, so that you don't need to load it everytime you call this function.